taipy/core/scenario/_scenario_manager.py

```diff
@@ -270,6 +270,24 @@ def _get_all_by_cycle(cls, cycle: Cycle) -> List[Scenario]:
     def _get_primary_scenarios(cls) -> List[Scenario]:
         return [scenario for scenario in cls._get_all() if scenario.is_primary]
 
+    @classmethod
+    def _sort_scenarios(
+        cls,
+        scenarios: List[Scenario],
+        descending: Optional[bool] = False,
```

Comment:
`descending` should not be an optional. Just a bool. We have some linter rules preventing from merging that are failing because of that.

Reply:
Hi @jrobinAV,

Did that update fix it?